Header header
uint16 ESP_21_CRC
uint8 ESP_21_BZ
int16 BR_Eingriffsmoment
uint8 ESP_Diagnose
uint8 ESC_v_Signal_Qualifier_High_Low
uint8 ESP_Vorsteuerung
uint8 OBD_Schlechtweg
uint8 OBD_QBit_Schlechtweg
float32 ESP_v_Signal
uint8 ASR_Tastung_passiv
uint8 ESP_Tastung_passiv
uint8 ESP_Systemstatus
uint8 ASR_Schalteingriff
uint8 ESP_QBit_v_Signal
uint8 ABS_Bremsung
uint8 ASR_Anf
uint8 MSR_Anf
uint8 EBV_Eingriff
uint8 EDS_Eingriff
uint8 ESP_Eingriff
uint8 ESP_ASP
uint8 ESC_Neutralschaltung
